lib/elixir/lib/kernel.ex

```diff
@@ -2570,6 +2570,44 @@ defmodule Kernel do
     end
   end
 
+  @doc """
+  Returns `true` if `term` is a non-struct map; otherwise returns `false`.
```

Comment:
```suggestion
  Returns `true` if `term` is a non-struct map, otherwise returns `false`.
```

Reply:
Another idea is: Returns true if term is a map that is not a struct (…)

Reply:
I like it!

Reply:
[9cfb27d](https://github.com/elixir-lang/elixir/pull/13534/commits/9cfb27d6512b8046586c7cb87e47423f9a07b530)